main/autogen/autoloader.cc

```diff
@@ -421,7 +421,7 @@ void AutoloadWriter::write(const core::GlobalState &gs, const AutoloaderConfig &
                 ENFORCE(child->qname.package);
                 auto namespaceName = child->qname.package->show(gs);
                 // this package name will include the suffix _Package on the end, and we want to remove that
-                constexpr int suffixLen = "_Package"sv.length();
+                constexpr int suffixLen = char_traits<char>::length("_Package");
```

Comment:
If you check the source code, this is exactly what the string_view constructor will call to populate it's internal `length` field.